src/Goteo/Model/Invest.php

```diff
@@ -2466,4 +2466,17 @@ public function getInvestOrigin(): ?InvestOrigin
             return null;
         }
     }
+
+    static public function hasInvestToReward(User $user, Reward $reward): bool
```

Comment:
This code would grant access to subscription-based rewards regardless of the subscription status. For those rewards we need to check if there's an invest that is up to date with the subscription.